discover/glossary.md

```diff
@@ -58,7 +58,7 @@ Transaction volume(TV) is often used interchangeably with data consume volume (D
 
 </details>
 
-<details>
+<details>initialEpochData
```

Comment:
It seems to me that this text "initialEpochData" was added by mistake here

Reply:
removed